ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.18__ocp-cnv-odf-4.18.yaml

```diff
@@ -65,6 +65,7 @@ tests:
             ]
         }
       FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp", "self-managed-lp", "cnv-lp"]'
```

Comment:
Can we use the Firewatch base config file here and update the FIREWATCH_CONFIG dict?